packages/editor/src/components/rich-text/style.scss

```diff
@@ -103,7 +103,7 @@
 
 		// On mobile and in nested contexts, the plus to add blocks shows up on the right.
 		// This padding makes sure it doesn't overlap text.
-		& + .editor-rich-text__tinymce {
+		& + .editor-rich-text__tinymce.wp-block-paragraph {
```

Comment:
I don't like that we target specific blocks in a generic component, what's the reason here? We could do the opposite if needed. Target the richText in the block.

Reply:
>  Target the richText in the block.

Do you mean move these styles into the paragraph block? 

Given we have a unique case for the _empty_ paragraph block, I think it's okay to make rules specific to that. But you could have a point in where the styles should live.

Reply:
yes, move to paragraph what's specific to paragraph and leave here what's generic to RichText.

Reply:
Good thoughts. Fixed it in https://github.com/WordPress/gutenberg/pull/12106/commits/0879d8a81d8370c4ee7b750717d6d4fab33ec9cb:

![screenshot 2018-11-20 at 15 35 46](https://user-images.githubusercontent.com/1204802/48780457-3c178480-ecda-11e8-8bb5-7f057720022c.png)